ruby-training/trainees/alan/test_katas/01/math_maddness/math_maddness.rb

```diff
@@ -0,0 +1,26 @@
+require 'rspec/autorun'
+
+class Aritmetic_operations
+    
+    def self.add(first_number, second_number)
+       first_number +  second_number
+    end
+    
+    def self.substraction(first_number, second_number)
+        first_number - second_number
+    end
+
+    def self.multiply(first_number, second_number)
+        first_number * second_number
+    end
+
+    def self.divide(first_number, second_number)
+        first_number / second_number
+    end
+end
+
+calculus = Aritmetic_operations.add(12, 4)
+calculus = Aritmetic_operations.substraction(15,5)
```

Comment:
Remember adding a space after commas that separate arguments!! (In this case you should remove this code, it's more of a reminder for the future and other cases).